### PR TITLE
filter documentation updated

### PR DESCRIPTION
**Pull request by devbysn**
Updated parameters and object in filter documentation!
issue - https://github.com/ldapjs/node-ldapjs/issues/909

Parameters :
{
  subInitial: 'foo',
  subAny: ['bar', 'cat'],
  subFinal: 'dog'
}

Object : 
const f = new SubstringFilter({
  attribute: 'cn',
  subInitial: 'foo',
  subAny: ['bar'],
  subFinal: 'baz'
});

### CODE REVIEW

File: docs/filters.md [hunk 1]
```diff
@@ -31,19 +31,19 @@ will be one of:
 # parseFilter(filterString)
 
 Parses an [RFC2254](http://www.ietf.org/rfc/rfc2254.txt) filter string into an
-ldapjs object(s).  If the filter is "complex", it will be a "tree" of objects.
+ldapjs object(s). If the filter is "complex", it will be a "tree" of objects.
 For example:
 
 ```js
-const parseFilter = require('ldapjs').parseFilter;
+const parseFilter = require("ldapjs").parseFilter;
```
Comment (jsumners) [inline docs/filters.md:38]: Please turn off your autocorrect tool and revert the changes not relevant to the issue.

**Review by jsumners — CHANGES_REQUESTED** [review]
We need to focus the PR on the specific issue and avoid tangential changes.

File: docs/filters.md (hunk 1 above)
Comment (devbysn) [reply docs/filters.md]: ohh, i didn't notice that.
corrected it, can you check?
